media/js/base/site.js

```diff
@@ -45,29 +20,17 @@
                 return 'linux';
             }
             if (pf.indexOf('MacPPC') !== -1) {
-                return 'oldmac';
-            }
-            if (/Mac OS X 10.[0-8]\D/.test(ua)) {
-                return 'oldmac';
+                return 'other';
             }
             if (pf.indexOf('iPhone') !== -1 ||
-                    pf.indexOf('iPad') !== -1 ||
-                    pf.indexOf('iPod') !== -1 ||
-                    pf.indexOf('MacIntel') !== -1 && 'standalone' in navigator) {
+                pf.indexOf('iPad') !== -1 ||
+                pf.indexOf('iPod') !== -1 ||
+                pf.indexOf('MacIntel') !== -1 && 'standalone' in navigator) { //iPadOS
                 return 'ios';
             }
-            if (ua.indexOf('Mac OS X') !== -1) {
+            if (ua.indexOf('Mac OS X') !== -1 && !/Mac OS X 10.[0-8]\D/.test(ua)) {
```

Comment:
OS X 10.0-8 made up ~2% of Mac visits over the past year. With this in mind, I think it's still worth serving them some conditional messaging.